fjord/mailinglist/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from fjord.mailinglist.models import MailingList
+
+
+class MailingListAdmin(admin.ModelAdmin):
+    list_display = ('name', 'members')
+
+
+admin.site.register(MailingList, MailingListAdmin)
```

Comment:
I don't know what I've been doing wrong, but I can't get the decorator to work. I use it, but it doesn't show up in the admin. I never spent the time to look into what was going on, though.